src/main.ts

```diff
@@ -1,11 +1,14 @@
 import { createApp } from "vue";
 import App from "@/App.vue";
 import { createSearchPlugin } from "@/SearchPlugin.ts";
-import router from "@/router/projectRoutes";
+import { router } from "@/router/projectRoutes";
 
 const app = createApp(App);
 app.use(router);
 
-createSearchPlugin(app, { router, getApi: "https://api.example.com" });
+// * TEMPORARY STATIC URL TO FETCH
+const staticApiUrl = "http://localhost:8080/api/resource/getResultByQueryTag";
```

Comment:
Let's pass it down, we want to start reducing hard coded solutions like this. For data initially yes but when it comes to API, we can't compromise on a static URL.

Reply:
deleted.